extensions/emmet/src/abbreviationActions.ts

```diff
@@ -90,45 +89,34 @@ export function wrapWithAbbreviation(args: any) {
 
 	function makeChanges(inputAbbreviation: string | undefined, previewMade?: boolean, definitive?: boolean): Thenable<any> {
 		if (!inputAbbreviation || !inputAbbreviation.trim() || !helper.isAbbreviationValid(syntax, inputAbbreviation)) {
-			return previewMade ? revertPreview(editor, rangesToReplace).then(() => { return false; }) : Promise.resolve();
+			return previewMade ? revertPreview(editor, rangesToReplace) : Promise.resolve();
```

Comment:
Returning false allows us to not revert the preview several times. That would not a problem per se, since revertPreview only replaces the current range with the original content, but why make extra unnecessary edits?